### ISSUE TITLE
MPC-HC won't snap in minimum or compact mode

### ISSUE THREAD

**Issue by RhinoFox** [digitalcreations/MaxTo#628]
When I use MPC-HC in the compact or minimal mode (Without Edges), the snap to a region doesn't work. In standard mode it does work. In the release version (2.01) it worked, but not in beta-5 version 2.1.0.697.

Play a movie in MPC-HC
Press: 1 (Minimum edges)
Try to snap with Left-mouse and Shift > No action
Press: 2 (Compact mode)
Try to snap with Left-mouse and Shift > No action
Press: 3 (Normal mode)
Try to snap with Left-mouse and Shift > It works

This was not an issue in 2.01 stable
I use Beta 5 because of the shutdown bug. I didn't see it anymore since beta-5

[config.zip](https://github.com/digitalcreations/MaxTo/files/5432675/config.zip)

**Comment by vegardlarsen** [comment]
@RhinoFox I assume you are testing this when dragging to move the window while holding Shift? Would it be an OK workaround to use the keyboard shortcuts instead?

**Comment by RhinoFox** [comment]
Yes with the Shift pressed.
Shortcuts are working. It's a workaround.

Are you solving the Shift option in the future?

**Comment by vegardlarsen** [comment]
I am not certain, and it most definitely won't get fixed for 2.1.0. We've made adjustments to which windows qualify for this for good reasons, and while I haven't tried MPC-HC, other programs will break if we allow this behavior with their borderless windows. How strict we are here is a trade-off, and we may adjust it again in the future.

I am closing this issue as this is not something we can change on a per-program basis, and using keyboard shortcuts is a workaround, but I have made a note of it.
